about/obs.js

```diff
@@ -56,8 +56,7 @@ exports.create = function (api) {
   })
 
   function get (id) {
-    if (!cache) {
-      cache = {}
+    if (Object.keys(cache).length === 0) {
```

Comment:
Your solution is pretty good, except I think it would be safer to create a `loaded = true` type check instead. Otherwise this could lead to multiple streams being created if get is called before the first stream returns an item.